tests/test_server.py

```diff
@@ -9,6 +9,7 @@
 from builtins import str
 
 import pytest
+from freezegun import freeze_time
```

Comment:
of course there's a package for this

Reply:
its python there is a package for everything 😍